src/mobile/android/settings.gradle

```diff
@@ -1,4 +1,8 @@
 rootProject.name = 'iotaWallet'
+include ':react-native-camera'
+project(':react-native-camera').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-camera/android')
```

Comment:
This is already imported [here](https://github.com/iotaledger/trinity-wallet/pull/781/files#diff-96e92bf7eb894a13ad69a927d2eb1a7cR23) and [here](https://github.com/iotaledger/trinity-wallet/pull/781/files#diff-96e92bf7eb894a13ad69a927d2eb1a7cR36)